www/content/tutorial.md

```diff
@@ -440,6 +440,12 @@ You can give `dbg` any expression you like, for example:
 dbg Str.concat singular plural
 ```
 
+You can also use `dbg` inside an expression: it will print its input to stderr and return it to the caller. For example:
+
+```roc
+inc = \n -> 1 + dbg n
+```
+
```

Comment:
Would it be worthwhile to describe what happens when you call `inc`? Something like

````suggestion
```roc
inc = \n -> 1 + dbg n
```

If we call this function as `inc (inc (inc 0))`, the value of `n` will be printed to stderr three times as `0`, `1`, and `2`, before the expression finally returns `3`.

````

Reply:
I also think it would be fine to do `inc = \n -> (dbg n) + 1`  and just not mention operator precedence. Not a big deal though.